cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,7 @@
+from django.core.management.base import BaseCommand
+import time
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        time.sleep(1)
```

Comment:
Does it really work???
What if 1 second is not enough for the database to start successfully?